uniffi_bindgen/src/bindings/kotlin/mod.rs

```diff
@@ -113,3 +95,29 @@ pub fn run_script(out_dir: &Path, script_file: &Path) -> Result<()> {
     }
     Ok(())
 }
+
+// Calculate the classpath string to use for testing
+pub fn classpath_for_testing(out_dir: &Path) -> Result<OsString> {
```

Comment:
I don't really love this system but it works okay.  I made #1169 to improve on it.